File: TypeScript/Directory/create-directory.ts
import * as fs from "fs"

function main(): void {
    // Create a directory.
    createDirectory("assets/remove/pa6KQGV9x68ksDc933Fok73e76X2WEq8")
}

main()

// Create a directory.
function createDirectory(path: string): void {
    fs.mkdirSync(path)
}

File: Dart/Directory/create_directory.dart
import "dart:core";
import "dart:io";

void main() {
  // Create a directory
  createDirectory("assets/remove/9WpmjE2856TAk3gNFKA5ypg5YL2b6D2r");
}

// Create a directory in a given path
void createDirectory(final String path) {
  Directory(path).createSync();
}
